source/geometry_model/interface.cc

```diff
@@ -36,6 +36,22 @@ namespace aspect
     {}
 
 
+    template <int dim>
+    void
+    Interface<dim>::update ()
+    {}
```

Comment:
You can delete this implementation, we do not need it anymore, because there is a default implementation in the base class of the interface (since the hackathon 2024, so quite recently).